docs/resources/vrf_fallback_route.md

```diff
@@ -0,0 +1,125 @@
+---
+# Documentation generated by "gen/generator.go"; DO NOT EDIT.
+# In order to regenerate this file execute `go generate` from the repository root.
+# More details can be found in the [README](https://github.com/CiscoDevNet/terraform-provider-aci/blob/master/README.md).
+subcategory: "Networking"
+layout: "aci"
+page_title: "ACI: aci_vrf_fallback_route"
+sidebar_current: "docs-aci-resource-aci_vrf_fallback_route"
+description: |-
+  Manages ACI VRF Fallback Route
+---
+
+# aci_vrf_fallback_route #
+
+Manages ACI VRF Fallback Route
+
+  -> This resource should not be used in combination with the `vrf_fallback_route_group` nested attributes of other resources for the same object. Doing so will result in unexpected behaviour.
```

Comment:
Do we mean that it should not be used with the vrf_fallback_routes attribute of the resource aci_vrf_fallback_route_group ?

Reply:
correct, this was the sentence we also use in the other resources like `aci_annotation`, can adjust if it is not clear

Reply:
adjusted